virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -250,6 +250,15 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [discrete]
 [id="virt-4-15-ki-virtualization"]
 ==== Virtualization
+
+// new for 4.15
+* A critical bug in `qemu-kvm` causes VMs to hang and experience I/O errors after xref:../../virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc#virt-hot-plugging-virtual-disks[disk hot plug] operations. This issue can also affect the operating system disk and other disks that were not involved in the hot plug operations. If the operating system disk stops working, the root file system shuts down. For more information, see link:https://access.redhat.com/solutions/7055333[Virtual Machine loses access to its disks after hot-plugging some extra disks] in the Red Hat Knowledgebase.
++
+[IMPORTANT]
+====
+Due to package versioning, this bug might reappear after updating {VirtProductName} from 4.13.z or 4.14.z to 4.15.
```

Comment:
nit? 4.15.0

We expect to fix this in 4.15.1.

Reply:
Fixed; the preview build will take a few minutes to update